src/main/java/client/Client.java

```diff
@@ -0,0 +1,54 @@
+package client;
+
+import io.qameta.allure.Step;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.builder.ResponseSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.specification.ResponseSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class Client {
```

Comment:
Perhaps class name PetClient would be more suitable

Reply:
But I have a PetClient as a subclass

Reply:
Sorry, I meant to say BaseClass :)